heron/ui/src/python/main.py

```diff
@@ -36,49 +35,68 @@ class Application(tornado.web.Application):
   ''' Application '''
 
   def __init__(self):
-    # TODO: Change these to query string parameters, since
+    # Change these to query string parameters, since
     # current format can lead to pattern matching issues.
-    # pylint: disable=line-too-long,bad-whitespace
     callbacks = [
         (r"/", handlers.MainHandler),
 
         (r"/topologies", handlers.ListTopologiesHandler),
-        (r"/topologies/filestats/([^\/]+)/([^\/]+)/([^\/]+)/([^\/]+)", handlers.ContainerFileStatsHandler),
-        (r"/topologies/([^\/]+)/([^\/]+)/([^\/]+)/config", handlers.TopologyConfigHandler),
-        (r"/topologies/([^\/]+)/([^\/]+)/([^\/]+)/([^\/]+)/([^\/]+)/exceptions", handlers.TopologyExceptionsPageHandler),
-        (r"/topologies/([^\/]+)/([^\/]+)/([^\/]+)", handlers.TopologyPlanHandler),
+        (r"/topologies/filestats/([^\/]+)/([^\/]+)/([^\/]+)/([^\/]+)",
```

Comment:
do you think callbacks can be defined as a list outside and assign this outside list to callbacks = listofcallbacks. This will provide space per line and you don't need to introduce new lines.